doc/news/changes.h

```diff
@@ -110,7 +110,17 @@ inconvenience this causes.
 <a name="specific"></a>
 <h3>Specific improvements</h3>
 
-<ol>
+<ol> 
+ <li> New: Add remove_hanging_nodes and remove_anisotropy in
```

Comment:
if you write these as `GridTools::remove_hanging_nodes()` and `GridTools::remove_anisotropy()`, Doxygen will automatically link this entry to the correct place in the documentation. Same below.